flavors/template-Exasol-all-python-3.12-conda/flavor_base/derived_from

```diff
@@ -0,0 +1 @@
+flavors/template-Exasol-all-python-3.10-conda
```

Comment:
?
```suggestion
flavors/template-Exasol-all-python-3.12-conda
```

Reply:
also supposed to be like that in `flavors/template-Exasol-all-python-3.12/flavor_base/derived_from`?

Reply:
Need to keep this file as it, `flavors/template-Exasol-all-python-3.10-conda` is correct here. (I know it's confusing).
This file is used for our package diff tool, it gives a hint where the original package versions are defined.